tools/build/build.sh

```diff
@@ -51,9 +51,9 @@ if (( $# > 0 )); then
   readonly GIT_ROOT=$(git rev-parse --show-toplevel)
   # Rather than a working directory of something like "/worker", mirror
   # the path on the host so that symlink tricks work as expected.
-  docker run --rm -ti -v "$GIT_ROOT":"$GIT_ROOT" -w "$GIT_ROOT" -e SENTRY_DSN=${SENTRY_DSN:-} $IMAGE_NAME "$@"
+  docker run --rm -i -v "$GIT_ROOT":"$GIT_ROOT" -w "$GIT_ROOT" -e SENTRY_DSN=${SENTRY_DSN:-} $IMAGE_NAME "$@"
```

Comment:
Why are you removing the `-t`? 

From https://docs.docker.com/engine/reference/run/#foreground:
"For interactive processes (like a shell), you must use -i -t together in order to allocate a tty for the container process."

Reply:
sorry, this was just a test. this is a draft branch so not the intended final version - i suppose I shouldn't have added you as reviewer yet, but need your insight on why other stuff was failing

the android check was complaining about it not being a TTY so, i removed this to see if that would fix it - it did. yet to do is to actually handle this case